test/src/Provider/AbstractProviderTest.php

```diff
@@ -245,10 +255,10 @@ public function testRandomGeneratorCreatesRandomState()
     {
         $xstate = str_repeat('x', 32);
 
-        $generator = m::mock('RandomLib\Generator');
+        $generator = m::mock(RandomGenerator::class);
         $generator->shouldReceive('generateString')->with(32)->times(1)->andReturn($xstate);
 
-        $factory = m::mock('RandomLib\Factory');
+        $factory = m::mock(randomFactory::class);
```

Comment:
> I'm not entirely convinced about using ::class inside of tests, it feels like change for the sake of change.

@shadowhand not at all. If we want to change the namespace for something, we don't have to refactor the entire file. Less code, without sacrificing clarity, which is easier to maintain: definitely not for the sake of change.